Storage/SQL/SQLiteHistory.swift

```diff
@@ -82,18 +82,30 @@ public class SQLiteHistory {
 
     private var ignoredSchemes = ["about"]
 
-    required public init(db: BrowserDB) {
+    required public init?(db: BrowserDB, version: Int? = nil) {
```

Comment:
I added one in here I think?

        for i in 0...BrowserTable.DefaultVersion {
            let history = SQLiteHistory(db: db, version: i)
            XCTAssertNotNil(history)
        }